### ISSUE TITLE
Show affixes on the item 

### ISSUE THREAD

**Issue by washow** [PathOfBuildingCommunity/PathOfBuilding#2744]
Currently, the mods on the item are simply shown as a list of mods and there is no way to tell which mods are prefix/suffix.
![GZ3xl23](https://user-images.githubusercontent.com/14829830/117249615-bb3f9b00-adf6-11eb-8ad0-4bb24fcb404c.png)

It would be so much easier to identify the mods if the mods divided into prefix/suffix instead of being combined as one.

**Comment by Regisle** [comment]
that is because we are not told if its a prefix or a suffix and there are some edge cases which makes it hard or impossible to tell which affix a mod belongs to, its possible to update item importing as the advanced copy paste from poe will tell us that info, but characters wont have that available

**Comment by asamcclellan** [comment]
I agree. Not only does this make the POB warriors go wild, but is not conducive to feeling comfortable building items without looking at Craftofexile.

I think we need conditionals here as well: display an alert if the item is impossible to make.

**Comment by Wires77** [comment]
This is solved by advanced copy/paste parsing in the next update
